packages/mysql-8.0-mroonga/debian/changelog

```diff
@@ -1,3 +1,9 @@
+mysql-8.0-mroonga (13.05-3) unstable; urgency=low
+
+  * Rebuilt against MySQL 8.0.35.
```

Comment:
mysql-8.0-mroonga (13.05-3) is built with MySQL 8.0.36.
So, we should write "Rebuilt against MySQL 8.0.36" in this change log.

Reply:
Good catch!

In general, we should generate the message automatically to prevent this type mistake.

Reply:
```
mysql --version
mysql  Ver 8.0.36-0ubuntu0.22.04.1 for Linux on x86_64 ((Ubuntu))
```

Oh. That's for sure.